vault.js

```diff
@@ -1,4 +1,19 @@
 'use strict';
-module.exports = function() {
-  
-};
\ No newline at end of file
+module.exports = function () {
+  let obj = {};
+  let setValue = function (key, value) {
+    if (!key) {
+      return null;
+    } else {
+      obj[key] = value;
+    }
+  }
+  let getValue = function (key) {
+    if (!key || !obj[key]) {
```

Comment:
Using the `.hasOwnProperty()` method will cover more situations. For example the key pass was 'toString' your `getValue` would return the definition of the `toString` method.